how-to/integrate-with-sso/integrate-with-okta/client/src/auth.ts

```diff
@@ -19,44 +9,84 @@ codeChallenge: should be a calculated value of Base64-URL-encoded SHA256 hash of
 Note: Okta access token are hard coded to expire in 60 minutes, when using the Okta auth servers.
 https://support.okta.com/help/s/article/What-is-the-lifetime-of-the-JWT-tokens?language=en_US#:~:text=When%20using%20the%20Okta%20authorization,Refresh%20Token%3A%20100%20days
 */
-const codeVerifier = "Zg6klgrnixQJ629GsawRMV8MjWvwRAr-vyvP1MHnB6X8WKZN";
-const codeChallenge = "iF_7prUeJ6rr3jMG3LmhW3R1cZ2ecZavFqS0jtb6tzo";
+const CODE_VERIFIER = "Zg6klgrnixQJ629GsawRMV8MjWvwRAr-vyvP1MHnB6X8WKZN";
```

Comment:
Should these things be hardcoded?

Reply:
I have moved them to config, but left ours in place, as they are a pain to generate, added the instruction to the readme on how to do so.